Updates to origin trials articles (#2569)
GoogleChrome/developer.chrome.com: site/en/blog/origin-trials/index.md

the [deprecation trial to continue using AppCache](/origintrials/#/view_trial/1776670052997660673) until Chrome 93.

## Take part in an origin trial

1. Choose an origin trial from the [list of active trials](https://developers.chrome.com/origintrials/#/trials/active).

> 1. Request a token (a string of letters and numbers) by clicking the **Register** button and

Reviewer: Does this definition matter to this audience? I'd guess the developers who will run these instructions know what to expect?

Author: You're right — I deleted ' (a string of letters and numbers)'.

filling out the form.

> 1. Provide the token on every web page for which you want the trial feature to be enabled, using one

Reviewer: ```suggestion
1. Provide the token to every web page participating in this trial, using one
```

Reviewer: The "who" for indicating participation is clear at the start, so it doesn't need to be stated. This modification slightly shortens and clears up your intent.

Author: Agree that this has become a bit wordy, but I'm uncomfortable with 'every web page participating'. (It's 'you', the developer, who is participating). Also — nit — the token is not provided 'to' the web page: it's made available on the web page 'to' the browser to enable the feature.

I've changed it to 'Provide the token on every web page for which you want the trial feature to be enabled: ...'

1. When your token expires, you will get an email with a renewal link.
   To do so, you are again asked to submit feedback.

### Provide a token programmatically {: #programmatic}

Instead of providing a token as an HTTP header or as a meta tag in the HTML of your page, as
described earlier, you can inject a token with JavaScript:

```javascript
const otMeta = document.createElement('meta');
otMeta.httpEquiv = 'origin-trial';
otMeta.content = 'TOKEN_GOES_HERE';
document.head.append(otMeta);
```

> This technique is used to provide tokens for [third-party trials](#third-party).

Author: It's tricky. The most common reason for providing a token via JavaScript  (not in a header or in HTML) is for third-party tokens, but I realised that sites may want to provide tokens via JavaScript for other reasons. I wanted to make this a bit clearer: you will need to use JavaScript to inject a token programmatically if you're a third party participating in an origin trial on multiple sites,  but you can also use JavaScript to inject tokens on your own sites if you prefer.

> This technique is used to provide tokens for [third-party trials](#third-party).

Author: I've left the format as-is but updated the text so the language is a bit more direct/active, to match your wording.

{% Aside 'warning' %}

Usually if an API lands unchanged after a successful origin trial, there is a
short period between the end of the origin trial and the date the implementation ships in the

> browser when the API will not be available. This is by design. If Chrome were to avoid the

Reviewer: ```suggestion
browser when the API is unavailable. This is by design. If Chrome were to avoid the
```
